distmap

```diff
@@ -7,7 +7,7 @@ use File::Which;
 use File::Temp;
 
 use FindBin qw/$RealBin/;
-use lib "$RealBin/bin";
+use lib "$RealBin/../lib/perl5";
 use Utility;
```

Comment:
By the way, I just realize: before I rebase into the latest master, you have to change other scripts in the bin folder to use the "lib/perl5", because they import the Utility package...